### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by sunzhaochang**
#### What type of PR is this?

/kind cleanup

#### Which issue(s) this PR fixes:

Fixes some typos

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/phases/kubelet/flags.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ type kubeletFlagsOpts struct {
 // (from lower to higher):
 // - actual hostname
 // - NodeRegistrationOptions.Name (same as "--node-name" passed to "kubeadm init/join")
-// - "hostname-overide" flag in NodeRegistrationOptions.KubeletExtraArgs
```
Comment (heqg) [inline cmd/kubeadm/app/phases/kubelet/flags.go:53]: looks good to me

another File Has the Similar Problem
https://github.com/kubernetes/kubernetes/pull/106998

**Comment by neolit123** [comment]
/lgtm
/approve
for kubeadm

**Comment by bowei** [comment]
Can we couple this with an actual bug fix for an issue?

**Pull request closed without merging**